src/UI/Component/Breadcrumbs/Breadcrumbs.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/* Copyright (c) 2017 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\UI\Component\Breadcrumbs;
+
+/**
+ * Interface for Breadcrumbs
+ * @package ILIAS\UI\Component\Breadcrumbs
+ */
+interface Breadcrumbs extends \ILIAS\UI\Component\Component {
+
+	/**
+	 * Get all crumbs.
+	 *
+	 * @return 	\ILIAS\UI\Component\Link\Standard[]
+	 */
+	public function getCrumbs();
```

Comment:
Please rename the entries to `items` completely, as we use this wording in the rules.